entrypoint.sh

```diff
@@ -80,12 +87,11 @@ then
       pre_tag="$initial_version"
     fi
 else
-    log=$(git log $tag..HEAD --pretty='%B')
+    log=$(git log $tag..HEAD --pretty='%B' || git log "v$tag..HEAD" --pretty='%B')
+    # get current commit hash for tag
+    tag_commit=$(git rev-list -n 1 $tag)
```

Comment:
I proposed here to first check if the tag is with or without prefix, e.g
```bash
if [[ -z "$(git --no-pager tag -l ${tag})" ]]; then
    tag=v${tag}
fi
log=$(git log $tag..HEAD --pretty='%B')
# get current commit hash for tag
tag_commit=$(git rev-list -n 1 $tag)
```

Reply:
We could also use this logic for the `pre_tag` for completeness and also to correct the display in line 150 of the $pre_tag. But this is not required to make the tagging working correctly.

Reply:
That might work, I did this as I did for the line above:
https://github.com/alexeyatbluescape/github-tag-action/commit/cd1ce0389a23a31bfb4453431c9f3dad6d36c725#diff-6f9d41d046756f0ddc2fcee0626bdb50100d12b88f293734eff742818e03efa2R92

Here is the run testing this scenario: https://github.com/alexeyatbluescape/github-tag-action/commit/cd1ce0389a23a31bfb4453431c9f3dad6d36c725#diff-6f9d41d046756f0ddc2fcee0626bdb50100d12b88f293734eff742818e03efa2L92

If I get bored I might re-write this whole thing in python or node...

Reply:
Looked more at the code, your suggestion will also work as it is only used in `semver` commands below, which will work with `v` or without it.